content/foundations/responsive-design.mdx

```diff
@@ -0,0 +1,116 @@
+
+- **Minimum viewport width**: 320px
+- **Minimum viewport height**: 256px
+
+Providing support at these smaller sizes enable people with low vision to use GitHub with a browser zoom enabled, up to 400% on a 1280px wide screen. [Read more about this accessibility requirement](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
+
+To understand how to break down a page to work on smaller viewports, check out [Responsive foundations](/foundations/layout#responsive-foundations) and [Responsive behavior](/foundations/layout#responsive-behavior) sections in the [Layout](/foundations/layout) page.
+
+<!--
+
+Todo: Link to Figma Templates
+
+> When designing a mobile-friendly rendering of a page, we recommend using a viewport width of **375px**, as this is the most common size among small smartphones. However, the page should still be tested and be fully functional at smaller widths for accessibility reasons, all the way to **320px**.
+
+-->
+
+### Pointing device
+
+Instead of trying to infer the input mechanism from screen size, use the device characteristics through [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries) to adapt the experience accordingly.
+
+The table below shows possible device scenarios according to `pointer` and `hover` media features:
+
+&nbsp;             | `pointer: coarse`                                | `pointer: fine`
+-------------------|:------------------------------------------------:|:---------------------:
+**`hover: none`**  | Smartphones, touch screens                       | Stylus-based screens
+**`hover: hover`** | Virtual reality headsets, video-game controllers | Mouse, touch pad
+
+#### Pointer types
+- A `coarse` pointer means the primary input mechanism isn’t very accurate. A finger on a touchscreen is a coarse pointer.
+
+  - When designing for coarse/touch, make sure that the target size is large enough to be easily tapped. [Read more about this accessibility requirement](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html).
+
+  - **Minimum coarse/touch target**: 44px
+
+
+- A `fine` pointer means the primary input mechanism supports accurate control. A mouse or a stylus is a fine pointer.
+
+  - **Recommended minimum fine/mouse target**: 20px
+
+<!--
+
+Todo: Touch-friendly strategy section.
+
+Examples of touch adaptation with out-of-boudaries touch targets, when applied to Button and IconButton components, following comparison with ActionListItem, which increases its height instead.
+
+Image source:
+https://www.figma.com/file/YlBTqHTbdRwYQt0bfIHvGw/Interface-guidelines?node-id=457%3A89625
+
+-->
+
+
+#### Hover support
+
+Browsers report if the primary input mechanism can hover over elements with the `hover` media feature.
+
+Devices that don’t support hovering such as smartphones and tablets may need adapted experiences so that the user can interact with the page without hovering over elements.
```

Comment:
@ericwbailey I wonder if that's a topic for an _Accessibility_ page? Our current [introduction](https://primer.style/design/foundations/introduction) covers these areas separately, but they are so intertwined indeed!

Reply:
I think it's worth mentioning here if only because there's the chance it'll remind folks who aren't thinking about accessibility that it is a consideration.